### PR TITLE
Added Twilio Bee

### PR DESCRIPTION
**Pull request by jamesbvaughan**
This Bee allows you to send SMS messages through Twilio's API.

It makes use of @carlosdp's Twilio Go helper library [twiliogo](https://github.com/carlosdp/twiliogo).

### CODE REVIEW

File: bees/twiliobee/twiliobee.go [hunk 1]
```diff
@@ -0,0 +1,72 @@
...
+func (mod *TwilioBee) Action(action bees.Action) []bees.Placeholder {
+	outs := []bees.Placeholder{}
+
+	switch action.Name {
+	case "send":
+		body := ""
+
+		action.Options.Bind("body", &body)
+
+		twilio.NewMessage(mod.client, mod.from_number, mod.to_number, twilio.Body(body))
```
Comment (muesli) [inline bees/twiliobee/twiliobee.go]: Idea: we could make ```to_number``` an option of this action, too. That would allow you to set different receivers for different events. (can be a separate pull request though)

Comment (jamesbvaughan) [inline bees/twiliobee/twiliobee.go]: Ahh, I like that idea. I'll do it in a separate pull request. Do you think it should be an option for the bee and the action (so that you could set a default) or just for the action?

Comment (muesli) [inline bees/twiliobee/twiliobee.go]: Would be nice to have both!

Comment (jamesbvaughan) [inline bees/twiliobee/twiliobee.go]: Cool. If I have time later today I'll do that, but otherwise I'll do it next week.

**Comment by muesli** [comment]
Nice work, thank you!

File: bees/twiliobee/twiliobee.go (hunk 1 above)
Comment (PedroMD) [inline bees/twiliobee/twiliobee.go]: I was just looking into this bee and my first thought was precisely (why isn't this part of the action ?!) 

Nice work so far - will wait for that PR :)

Comment (jamesbvaughan) [inline bees/twiliobee/twiliobee.go]: Ahh thanks for the bump. I had almost forgotten about this!
